draft-ietf-acme-acme.md

```diff
@@ -1867,7 +1867,7 @@ type (required, string):
 
 token (required, string):
 : A random value that uniquely identifies the challenge.  This value MUST have
-at least 128 bits of entropy, in order to prevent an attacker from guessing it.
+at least 128 bits of entropy.
```

Comment:
This same change should be made on [line 1972](https://github.com/ietf-wg-acme/acme/pull/284/files#diff-8430e2aa241beb4ac49b252db20d4ee8R1972) and [line 2074](https://github.com/ietf-wg-acme/acme/pull/284/files#diff-8430e2aa241beb4ac49b252db20d4ee8R2074).